step02/presigned_url_api.yaml

```diff
@@ -0,0 +1,93 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Thumbnail Service API
+  license:
+    name: MIT
+servers:
+  - url: http://lolcats.gg/v1/
+paths:
+  /users/me:
+    get:
+      tags:
+        - users
+      summary: Gets profile of current user
+      operationId: app.get_profile
+      responses:
+        '200':
+          description: User Profile
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        default:
+          $ref: '#/components/responses/GenericError'
+  /users/presigned-url:
+    get:
+      tags:
+        - presigned url
+      summary: returns a presigned url for user  to upload image
+      parameters:
+        - $ref: '#/components/parameters/contentTypeParam'
```

Comment:
let's get these query params implemented without using $refs. you should be able to put the parameters directly in here